common/djangoapps/third_party_auth/middleware.py

```diff
@@ -45,12 +47,11 @@ def process_view(self, request, view_func, view_args, view_kwargs):  # pylint: d
         collection of user consent for sharing data with a linked third-party
         authentication provider.
         """
-        running_pipeline = request.session.get('partial_pipeline')
-
-        if not running_pipeline:
+        if not pipeline.running(request):
             return
 
         view_module = view_func.__module__
-        quarantined_modules = request.session.get('third_party_auth_quarantined_modules', None)
+        strategy = social_utils.load_strategy(request)
+        quarantined_modules = strategy.session_get('third_party_auth_quarantined_modules')
```

Comment:
@bradenmacdonald That's right, it does the same thing, but I reckoned that if for some reason the technique for getting something from the session changed in the future and PSA took care of it through `session_get`, our use of the API would save us having to manually change it ourselves. Not sure if this is a valid concern though.